packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -172,7 +171,13 @@ export default class Node extends PureComponent {
         const hasTimeableNodeVisibility = node?.properties?._hasTimeableNodeVisibility;
 
         if (hasTimeableNodeVisibility) {
-            const circleColor = isDisabled ? 'error' : 'primaryBlue';
```

Comment:
The internal logic of timable node visibility has changed. That's why this also has changed. The state is rather depending on the actual visibility state (disabled/enabled) than the "calculated" state based on the datetime.

https://github.com/neos/timeable-node-visibility

Reply:
I guess so. Would be a feature and we would need some idea how we want to display that.

Reply:
But I think your solution does not cover all scenatios atm. Please check the tests in the `neos/timeable-node-visibility` package: https://github.com/neos/timeable-node-visibility/blob/9.0/Tests/Behavior/Features/HandleExceeded.feature

Reply:
And the backend implementation to determine the state: https://github.com/neos/timeable-node-visibility/blob/9.0/Classes/Service/TimeableNodeVisibilityService.php#L129-L154